students/nam_vo/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,44 @@
+def exchange_first_last(seq):
+    """Return a copy of seq with the first and last items exchanged"""
+    return (seq[-1:] + seq[1:-1] + seq[:1])
+
+def remove_every_other(seq):
+    """Return a copy of seq with every other item removed"""
+    return seq[1::2]
+
+def remove_first_last_4(seq):
+    """Return a copy of seq with the first 4 and the last 4 items removed, and then every other item in the remaining sequence"""
+    first_4_removed_seq = seq[4:]
+    last_4_removed_seq = first_4_removed_seq[:-4]
+    every_other_seq = last_4_removed_seq[::2]
+    return every_other_seq
+
+def reverse_element(seq):
+    """Return a copy of seq with the elements reversed (just with slicing)"""
+    return seq[::-1]
+
+def slice_third(seq):
+    """Return a copy of seq with the last third, then first third, then the middle third in the new order"""
+    return (seq[-3:-2] + seq[2:3] + seq[5:6])
```

Comment:
this is incorrect - if your input is (1, 2, 3, 4, 5, 6, 7, 8, 9) then expected output is: (7, 8, 9, 1, 2, 3, 4, 5, 6)